lib/psm.c

```diff
@@ -511,13 +511,14 @@ void rpmpsmNotify(rpmpsm psm, int what, rpm_loff_t amount)
  */
 static void markReplacedInstance(rpmts ts, rpmte te)
 {
+    /* this must match rpmNameVersionCompare in depends.c */
```

Comment:
No, please don't, the comment is correct. We do have rpmNameVersionCompare in depends.c. See the nameversioncompare.diff patch...